README.md

```diff
@@ -1,19 +1,24 @@
 # Butler
-The beginnings of a Sculpin-based style guide creation tool. 
+A tool to automate front-end development tasks and streamline prototyping.  
 
 ## Dependencies
 1. NodeJS 
    1. To intall NodeJS using Homebrew run `brew install node` 
 1. [Sculpin](https://sculpin.io/getstarted/)
-    1. `curl -O https://download.sculpin.io/sculpin.phar`
-    2. `chmod +x sculpin.phar`
-    3. `mv sculpin.phar sculpin` or if you are using vagrant run `sudo -H mv sculpin.phar /bin/sculpin` and skip step 4
-    4. `mv sculpin ~/bin/` 
+    
+     By default Butler expects your project to include Sculpin as a Composer dependency. 
+     You can change this by updating the `defaults.sculpin_run` setting. If you're using a global Sculpin install the `defaults.sculpin_run` variable should be set to `sculpin`.
 
 ## Add Butler to a project
 1.   If the project does not already have npm dependencies run `npm init` to create a `package.json` file
 
 1.   Add Butler as a dependency by running `npm install --save palantirnet/butler`
+     
+     When Butler is installing it will ask for the GitHub repository for your project. Please provide the HTTPS link for the project repository.
+     Example: `https://github.com/palantirnet/[project].git`
```

Comment:
If you didn't use the `.git` part on MCOR I would say that it doesn't require it. This is the example that I use in the prompt for it though.